pgtap/lineGraph/lineGraphFull/lineGraphFull-rows-check.sql

```diff
@@ -0,0 +1,45 @@
+\i setup.sql
+
+SELECT plan(10);
+
+SET extra_float_digits = -3;
+
+-- Check whether the same set of rows is always returned
+
+PREPARE expectedOutput AS
+SELECT * FROM pgr_lineGraphFull(
+    'SELECT id, source, target, cost, reverse_cost
+    FROM edge_table
+    ORDER BY id'
+);
+
+PREPARE descendingOrder AS
+SELECT * FROM pgr_lineGraphFull(
+    'SELECT id, source, target, cost, reverse_cost
+    FROM edge_table
+    ORDER BY id DESC'
+);
+
+PREPARE randomOrder AS
+SELECT * FROM pgr_lineGraphFull(
+    'SELECT id, source, target, cost, reverse_cost
+    FROM edge_table
+    ORDER BY RANDOM()'
+);
+
+SELECT set_eq('expectedOutput', 'descendingOrder', '1: Should return same set of rows');
```

Comment:
../../pgtap/lineGraph/lineGraphFull/lineGraphFull-rows-check.sql                                     (Wstat: 0 Tests: 10 Failed: 10)
  Failed tests:  1-10
.Martk the failed tests with todo